index.html

```diff
@@ -6,10 +6,176 @@
+                <li><a href="./about.html">Contact</a></li>
+            </ul>
+        </div>
+        <div class="desk-header-menu justify-content-around">
+            <a href="./index.html"><img class="header-logo" src="./image/headerLogo.png" alt="logo"></a>
+            <div class="desktop-menu d-flex">
+                <div class="menu-desktop d-flex gap-5 p-4">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#main-program">Program</a>
+                    <a href="#main-program">Join</a>
+                    <a href="#partner">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="./about.html">Contact</a>
+                </div>
+            </div>
+        </div>
+        <div class="top container-fluid pt-4 p-4">
+            <header class="header position-fixed">
+                <div class="bar" id="bar1"></div>
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                
+                <p class="border border-3 border-white">Welcome to the Carbon Capture Summit 2022, where industry leaders will meet in Amsterdam to explore next-generation technologies for carbon capture, utilisation, storage, and transportation to meet net-zero emissions targets.</p>
+                <h2 class="text-dark">2022.11.24(THR) ~ 25(FRI)</h2>
+                
+                <h5>National Museum, Van Der Valk, Amsterdam, Netherlands</h5>
+            </main>
+        </div>
+        <div class="" id="main-program">
+            <h3 class="text-white text-center pt-4">Main Program</h3>
+            <h3 class="text-danger text-center">____</h3>
+            <div class="mpcontents">
+                <div class="main-pogram-content justify-content-between" id="lecture">
+                    <a href=""><img class="" src="./image/lecture.png" alt="lecture logo"></a>
+                        <h5 class=" program-info text-danger font-weight-bold w-25">lecture</h5>
+                    <p class="program-details w-50 text-white m-0">Listen the speakers from various countries abouth the message of sharing and opening.</p>
+                </div>
+                <div class="main-pogram-content  justify-content-between" id="cc-exhibition">
+                    <a href=""><img class="" src="./image/cc_exhibition.png" alt="cc-exhibition logo"></a>
+                    <h5 class="program-info text-danger font-weight-bold w-25">CC Exhibition</h5>
+                    <p class="program-details w-50 text-white m-0">Appreciate various creating applying CC liscence of artists, Organize from art center nabi.</p>
+                </div>
```

Comment:
- [ ] Kindly use `article` tags to create both your speaker and program cards since they hold semantic meaning unlike `div`. This will improve the accessibility and SEO score of your website a lot.

Reply:
Changed the tag to article.